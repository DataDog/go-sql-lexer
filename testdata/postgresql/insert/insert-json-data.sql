INSERT INTO events (data) VALUES ('{"type": "user_signup", "user_id": 1}');
