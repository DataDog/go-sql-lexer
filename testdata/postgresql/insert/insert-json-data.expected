[
    {
        "expected": "INSERT INTO events ( data ) VALUES ( ? )",
        "statement_metadata": {
            "size": 12,
            "tables": [
                "events"
            ],
            "commands": [
                "INSERT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]